moveit_core/CHANGELOG.rst

```diff
@@ -2,6 +2,53 @@
 Changelog for package moveit_core
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+2.4.0 (2022-01-20)
+------------------
+* Move background_processing (`#997 <https://github.com/vatanaksoytezer/moveit2/issues/997>`_)
+* Fix boost linking errors for Windows (`#957 <https://github.com/vatanaksoytezer/moveit2/issues/957>`_)
+* Delete backtrace hack (`#995 <https://github.com/vatanaksoytezer/moveit2/issues/995>`_)
+* Use size_t for index variables (`#946 <https://github.com/vatanaksoytezer/moveit2/issues/946>`_)
+* Remove moveit_build_options
+* Merge https://github.com/ros-planning/moveit/commit/f3ac6070497da90da33551fc1dc3a68938340413
+* Replace NULL with nullptr (`#961 <https://github.com/vatanaksoytezer/moveit2/issues/961>`_)
+  * Fixes `#841 <https://github.com/vatanaksoytezer/moveit2/issues/841>`_
+* Merge https://github.com/ros-planning/moveit/commit/a0ee2020c4a40d03a48044d71753ed23853a665d
```

Comment:
This really comes down to how clean we want the CHANGELOGS to be. The multiple sync merge commits should be removed. Also most packages have `moveit_build_options()` which didn't apply for MoveIt 2 and has been removed. Also, you apparently have your own GitHub account as upstream remote, so all the links are pointing to your repo instead of `ros-planning`. At least the last point should be fixed before merging

Reply:
Links should be good now.